Scripts/modelsystem.py

```diff
@@ -538,10 +541,21 @@ def _add_internal_demand(self, previous_iter_impedance, is_last_iteration):
             for tour in person.tours:
                 self.dtm.add_demand(tour)
         if is_last_iteration:
+            random.seed(zone_param.population_draw)
             self.dm.incmod.predict()
+            random.seed(None) 
+            line = "\t".join([
+                "person_id", "age_group", "gender", "car_user", "income",
+                "number", "area", "municipality", "nr_tours",
+                "total_access", "sustainable_access", "car_access"
+                ])
```

Comment:
Maybe move this to `Person` as class variable?

Reply:
Good idea. 

I wasn't happy having two list definitions even in same class, so decided to refactor fixed lists from `Person.__str__() `.